pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file:
```

Comment:
I was thinking something simple like "Interactively create a composer.json file defining the project's dependencies:".